toolkit/resources/manifests/package/update_manifests.sh

```diff
@@ -197,7 +197,6 @@ remove_packages_for_pkggen_core () {
     sed -i '/tdnf-python/d' $TmpPkgGen
     sed -i '/util-linux-lang/d' $TmpPkgGen
     sed -i '/wget/d' $TmpPkgGen
-    sed -i '/which/d' $TmpPkgGen
```

Comment:
Removing this line will add both `which` and `which-debuginfo`. We should change it to `which-debuginfo` instead.

Reply:
in update_manifests.sh, it already contains a section of code to remove *-debuginfo packages from pkggen_core_*.txt. Refer to Line 310 of update_manifests.sh